CMakeLists.txt

```diff
@@ -83,11 +113,43 @@ add_compile_definitions(PAGER_ENV="LESS=FRX LV=-c"
 						BINDIR="bin"
 						GIT_BUILT_FROM_COMMIT="")
 
-set(FALLBACK_RUNTIME_PREFIX /home/$ENV{USER})
-add_compile_definitions(FALLBACK_RUNTIME_PREFIX="${FALLBACK_RUNTIME_PREFIX}")
+if(WIN32)
+	set(FALLBACK_RUNTIME_PREFIX /mingw64)
```

Comment:
I am still wondering how we can deal with this properly. After all, we want Visual Studio users to be able to build a 32-bit version of Git, too.

Reply:
we can specify an option for 32 bit builds.
For visual studio we have an architecture option with cmake which can be set to x86.